validator/client/src/main/java/tech/pegasys/teku/validator/client/restapi/ValidatorTypes.java

```diff
@@ -21,12 +21,55 @@
 import tech.pegasys.teku.infrastructure.restapi.types.CoreTypes;
 import tech.pegasys.teku.infrastructure.restapi.types.DeserializableTypeDefinition;
 import tech.pegasys.teku.infrastructure.restapi.types.SerializableTypeDefinition;
+import tech.pegasys.teku.validator.client.restapi.apis.schema.ActiveValidator;
 import tech.pegasys.teku.validator.client.restapi.apis.schema.DeleteKeyResult;
 import tech.pegasys.teku.validator.client.restapi.apis.schema.DeleteKeysRequest;
 import tech.pegasys.teku.validator.client.restapi.apis.schema.DeleteKeysResponse;
 import tech.pegasys.teku.validator.client.restapi.apis.schema.DeletionStatus;
+import tech.pegasys.teku.validator.client.restapi.apis.schema.ImportStatus;
+import tech.pegasys.teku.validator.client.restapi.apis.schema.PostKeyResult;
+import tech.pegasys.teku.validator.client.restapi.apis.schema.PostKeysRequest;
 
 public class ValidatorTypes {
+
+  static SerializableTypeDefinition<PostKeyResult> POST_KEY_RESULT =
+      SerializableTypeDefinition.<PostKeyResult>object()
+          .name("PostKeyResult")
+          .withField(
+              "status",
+              DeserializableTypeDefinition.enumOf(ImportStatus.class),
```

Comment:
I wonder if we should use a static import for `enumOf` (and probably `listOf` elsewhere).  May make this read better. Actually even `STRING_TYPE` might be better as a static import.